src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -679,6 +696,19 @@ internal static partial class DbConnectionStringKeywords
         internal const string WorkstationID = "Workstation ID";
         internal const string ConnectRetryCount = "ConnectRetryCount";
         internal const string ConnectRetryInterval = "ConnectRetryInterval";
+
+        // Retry Logic
+        internal const string RetryStrategy = "RetryStrategy";
```

Comment:
These are so many connection string options. And they dont seem descriptive enough for anyone to look at the keyword and get an idea of where these might apply. Are these meant to be for connections only ? Should they be prefixed with Connect?

Reply:
I agree there are many, but effectively different retry strategies have different attributes. For example, for "FixedInterval" you only need a "RetryInterval" attribute, while for "Incremental" you need a "RetryIncrement" as well, and for "ExponentialBackoff" you need to specify min, max and delta. On purpose i was not prefixing them with Connect to avoid any confusion with existing Connection Resiliency parameters.